Packs/SysAid/Integrations/SysAid/SysAid.py

```diff
@@ -784,19 +881,95 @@ def service_record_delete_command(client: Client, args: Dict[str, Any]) -> Comma
     return command_results
 
 
+def service_record_attach_file_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    sr_id = str(args.get('id'))
+    file_id = str(args.get('file_id'))
+
+    response = client.service_record_attach_file_request(sr_id, file_id)
+    if response.status_code == 200:
+        msg = f'File uploaded to Service Record {sr_id} successfully.'
+    else:
+        msg = f'Error {response.status_code} occurred while uploading file to service record {id}.'
```

Comment:
It there some more information about the error you can add from the `response`?

Reply:
API documentation doesn't provide details on error responses.